app/context_processors.py

```
@@ -379,23 +379,130 @@ def delete_world_map():
+        cache_key, (participants_id_list, participants_mode_list), timeout=None
+    )
+
+    return participants_id_list, participants_mode_list
+
+
+# MARK: 初期化タスク
+def initialize_background_tasks(IS_LOCAL):
+    """
+    バックグラウンドタスクを初期化して同期的に必要なコンテンツを取得します。
+
+    この関数は以下を行います：
```

Ruff (0.14.8)

Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)